framework/test/play/data/validation/ValidationTest.java

```diff
@@ -6,6 +6,7 @@
 import java.util.Properties;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import play.ConfProperties;
```

Comment:
I had to check that `import java.util.Properties;` can be removed here and for the other test!

Reply:
Yeah, these `import java.util.Properties;` imports are now unused.